.github/workflows/validate-docs-links.yml

```diff
@@ -1,5 +1,6 @@
 name: "Validate Doc Links"
 on:
+  workflow_dispatch:
```

Comment:
What does this line do?

Reply:
It allows us to manually run the workflow from the GitHub UI by pressing a button. 

Currently, the linting action is only triggered on a pull request if a MDX file is updated. I added the ability to run it anytime. It's out of scope for this PR so I'll remove it.

Reply:
allows to run the workflow manually. out of scope, will remove